Add additional proximity sensors (#31236)
home-assistant/home-assistant.github.io: source/_integrations/proximity.markdown

- `towards`
- `unknown`

<div class="note">
To calculate the distance and the direction of travel for a tracked device or person, they must specify a geo-location.
</div>

### Nearest device

A sensor is created showing the device or person which is nearest (_shortest distance_) to the monitored zone. If several devices or persons are at the same nearest distance, this sensor displays them all.

> ### Nearest Distance

Reviewer: ```suggestion
### Nearest distance
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization) also in headings

> A sensor is created showing the distance of the nearest device or person to the monitored zone in a unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection. When a tracked person or device enters the monitored zone, the distance is set to 0.

Reviewer: ```suggestion
This sensor shows the distance of the device or person nearest to the monitored zone. The unit depends on your settings for the [Home Assistant unit system](/docs/configuration/basic). When a tracked person or device enters the monitored zone, the distance is set to 0.
```

### Nearest Direction of travel

> A sensor is created showing the direction of travel of the nearest device or person to the monitored zone. Possible states are:

Reviewer: ```suggestion
This sensor shows the direction of travel of the nearest device or person to the monitored zone. Possible states are:
```
